src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+| Open to anyone              | You must apply to use financial services                 |
+
+<ButtonLink to="/dapps/?category=finance">Explore defi apps</ButtonLink>
+
+## What can you do with defi?
+
+There's a decentralized alternative to most financial services. But Ethereum also creates opportunities for creating financial products that are completely new. This is an ever-growing list.
+
+- [Send money around the globe](#send-money)
+- [Stream money around the globe](#stream-money)
+- [Access stable currencies](#stablecoins)
+- [Lend/borrow funds](#lending)
+- [Borrow without collateral](#flash-loans)
+- [Start crypto savings](#savings)
+- [Swap tokens](#swaps)
+- [Trade??](#trading)
+- [Grow your portfolio](#investing)
+- [Fund your ideas](#crowdfunding)
+
+### Send money around the globe quickly {#send-money}
+
+As a blockchain, Ethereum is designed for sending transactions in a secure and global way. Payments go directly from one account to another in a matter of minutes (usually). To send/receive payments you'll need a wallet.
+
+<DocLink to="/wallets/" title="More on wallets" />
+
+#### Stream money around the globe... {#stream-money}
+
+It's also possible to stream money over Ethereum. This would let you pay someone their salary by the second, giving them access to their money whenever they need it.
+
+### Access to stable currencies {#stablecoins}
+
+Although a lot of cryptocurrencies are known for their volatility, the defi community has invested a lot of time in creating stability out of volatility with stablecoins. Their value stays pegged to an another asset, usually a popular currency like dollars.
+
+Coins like Dai or USDC have a value that stays within a few cents of a dollar. This makes them perfect for earning or retail. So far many people in Latin America have used stablecoins as a way of protecting their savings in a time of great uncertainty with their government-issued currencies.
+
+<DocLink to="/stablecoins/" title="More on stablecoins" />
+
+### Lending and borrowing {#lending}
+
+Today, lending and borrowing money all revolves around the individuals involved. Banks need to know whether you're likely to repay a loan before lending. Some see this as an invasion of privacy.
+
+Decentralized lending works without either party having to identify themselves. Instead the borrower must put up collateral that the lender will automatically receive if their loan is not repaid.
+
+Loans can be peer-to-peer, meaning a borrower will borrow directly from a specific lender. Or they can be pool-based where lenders provide liquidity to a pool that borrowers can borrow from.
+
+Some lenders even accept NFTs as collateral. NFTs are a deed to a unique asset, like a painting. [More on NFTs](/nfts/)
+
+A more experimental form of decentralized lending lets you borrow without collateral or providing any personal information...
+
+#### Flash loans {#flash-loans}
```

Comment:
Pretty solid explanation of a complex topic 😃 

If we want to add a higher level them to this, I see flash loans as the democratization of capital / removal of many capital requirements. e.g. in your example, typically to make money doing arbitrage, you need to have a ton of money for it to be worthwhile. Flash loans allow anyone to profit from these opportunities that are currently reserved for the privileged/wealthy.